data/units.yaml

```diff
@@ -0,0 +1,6 @@
+gravimetric:
```

Comment:
I think we agreed on a type schema file like:
```yaml
NUTS_region:
  id: string
  laBEL: string # short label for the UI, e.g. UKF2 NUTS2 region
  description: string # long description for the UI, e.g. Leicestershire, Rutland and Northamptonshire, UKF2 NUTS2 region
  level: (int or string?) # e.g. 2
  year_spec: int # year of specification release
  year_enforce|adoption: int # year of spec being enforced/adpopted
  boundary_URL: URL of the shapefile
units:
  gravimetric:
    label: Gravimetric Units
    unit: ms^-2
  euros_millions:
    label: Euros (Millions)
    unit: €m
...
```

Reply:
ah I see that is supposed to be defined in #106, feel free to add that PR and close 2 issues at once!

Reply:
Yes, though I haven't got to the NUTS schema yet, so made a separate units one in this PR. Does it not make more sense to have the geography schema in a separate file to the units schema?